sdk/deviceupdate/azure-iot-deviceupdate/CHANGELOG.md

```diff
@@ -1,15 +1,21 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0 (Unreleased)
```

Comment:
nit: update the release date. The release pipeline will block the release if the date is invalid.
```suggestion
## 1.0.0 (2022-09-02)
```